package.json

```diff
@@ -5,9 +5,9 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "downloadjs": "^1.4.7",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "react-redux": "^6.0.0",
+    "react": "^16.8.6",
```

Comment:
is there a reason you need to bump the react version?

Reply:
Noted, I'll change it back